website/docs/r/bigquery_dataset.html.markdown

```diff
@@ -73,6 +84,41 @@ The following arguments are supported:
 
 * `labels` - (Optional) A mapping of labels to assign to the resource.
 
+* `access` - (Optional) An array of objects that define dataset access for
+    one or more entities. Structure is documented below.
+
+The `access` block supports:
+* `role` - (Required unless `view` is set) Describes the rights granted to
+    the user specified by the other member of the access object. The following
+    string values are supported: `READER`, `WRITER`, `OWNER`.
+
+* `domain` - (Pick one) A domain to grant access to.
```

Comment:
I'm not quite sure what "Pick one" means in this context, do you mean Optional?

Reply:
I was mimic-ing the [BQ API documentation](https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#access), it basically means you have to pick exactly one of `domain`, `group_by_email`, `special_group`, `user_by_email`, or `view`. If there's a way to be more clear, I'm happy to change it.

Reply:
Ah, got it. I'd just try to match the rest of the Terraform documentation, where we just say Required or Optional there. I think that Pick One itself is pretty redundant, right? There isn't a way to specify multiple of the same field, so you don't really have a choice but to pick one (or zero, but that's covered in Required vs Optional).

Reply:
It's a little more nuanced because these are keys in a map, not values that can be taken by a single field. I updated the docs to reflect that (while marking these optional). Let me know what you think.